### ISSUE TITLE
Plugin sucrase is not resolving directories

### ISSUE THREAD

**Issue by Vehmloewff** [rollup/plugins#239]
- Rollup Plugin Name:  sucrase
- Rollup Plugin Version: 3.0.0
- Rollup Version: 1.32.0
- Operating System (or Browser): Ubuntu 19.10
- Node Version: 12.16.1

### How Do We Reproduce?

1. See the [repl](https://repl.it/@Vehmloewff/rollup-plugin-sucrase-no-resolve)
2. Click "run"
3. Yikes!

### Expected Behavior

It is expected that sucrase will resolve `./src` to `./src/index.ts` because `./src` is a directory.

### Actual Behavior

```
input.ts → output/bundle.js...
[!] Error: Could not resolve './src' from input.ts
Error: Could not resolve './src' from input.ts
    at error (/home/runner/rollup-plugin-sucrase-no-resolve/node_modules/rollup/dist/rollup.js:5365:30)
    at ModuleLoader.handleMissingImports (/home/runner/rollup-plugin-sucrase-no-resolve/node_modules/rollup/dist/rollup.js:12249:17)
    at ModuleLoader.<anonymous> (/home/runner/rollup-plugin-sucrase-no-resolve/node_modules/rollup/dist/rollup.js:12140:30)
    at Generator.next (<anonymous>)
    at fulfilled (/home/runner/rollup-plugin-sucrase-no-resolve/node_modules/rollup/dist/rollup.js:41:28)

exit status 1
```

**Comment by shellscape** [comment]
If you use surcase outside of Rollup, does it resolve the directories as expected?

**Comment by Vehmloewff** [comment]
Yes

```shell
▶ npx sucrase ./src -d ./dist --transforms typescript,imports         
src/index.ts -> dist/index.js
src/interfaces.ts -> dist/interfaces.js
src/lib/tracker.ts -> dist/lib/tracker.js
```

The resolve part for this plugin is done entirely 'in-house'.

https://github.com/rollup/plugins/blob/master/packages/sucrase/src/index.js#L14-L22

**Comment by larrybotha** [comment]
@Vehmloewff this should have been resolved by #390

**Comment by shellscape** [comment]
Just published this morning. I'll close this one.
